build/i2c/src/lib.rs

```diff
@@ -261,6 +326,7 @@ impl ConfigGenerator {
         let mut s = &mut self.output;
 
         assert!(self.disposition != Disposition::Devices);
+        assert!(self.disposition != Disposition::Sensors);
```

Comment:
So, under what circumstances should future Disposition values get added here? Should this be a match to ensure that people don't forget to extend it?

Reply:
Yes, good call.